File: 8_kyu/Twice_as_old.go
package kata

func TwiceAsOld(dadYearsOld, sonYearsOld int) int { 
  // calculate years ago
  delta := 2 * sonYearsOld - dadYearsOld
  if delta < 0 {
    // return years in the future also as positive number
    delta = -delta
  }
  return delta
}

File: 8_kyu/Twice_as_old.cs
using System;

namespace Solution {
    public class TwiceAsOldSolution {
        // Сalculate how many years ago father was 2x as old as son.
        // Return years in the future also as positive number.
        public static int TwiceAsOld(int dadYears, int sonYears)
            => Math.Abs(dadYears - 2 * sonYears);
    }
}
